NerdyPy/modules/admin.py

```diff
@@ -16,6 +24,65 @@ def __init__(self, bot):
         bot.log.info(f"loaded {__name__}")
 
         self.bot = bot
+        self.settings = {}
+        self._role_checker.start()
+
+    def cog_unload(self):
+        self._role_checker.cancel()
+
+    @tasks.loop(time=times)
+    async def _role_checker(self):
+        with self.bot.session_scope() as session:
+            for guild in self.bot.guilds:
+                self.bot.log.info(guild.name)
+                configuration = RoleChecker.get(guild.id, session)
+                if configuration is None:
+                    continue
+                if configuration.Enabled:
+                    for member in guild.members:
+                        if len(member.roles) == 1:
+                            if configuration.KickAfter > 0:
```

Comment:
kann mit in Zeile 41

`if configuration.Enabled and configuration.KickAfter > 0:`